cmd/rook/rook/rook.go

```diff
@@ -217,3 +217,14 @@ func TerminateFatal(reason error) {
 
 	os.Exit(1)
 }
+
+// GetOperatorBaseImageCephVersion returns the Ceph version of the operator image
+func GetOperatorBaseImageCephVersion(context *clusterd.Context) string {
+	output, err := context.Executor.ExecuteCommandWithOutput("ceph", "--version")
+	if err != nil {
+		logger.Errorf("failed to discover ceph base image version. %v", err)
```

Comment:
I don't know why this would ever fail, but should we return the error anyway?

Reply:
We just log it, isn't that enough? I don't want to fail the deployment for this.

Reply:
I'd suggest we return the error, and allow the caller to decide if they want to ignore it. In this case, I agree the caller would just log the error and assume it's octopus or newer.

Reply:
Oh I see sounds good.